src/styles/Utils/_variables.scss

```
@@ -0,0 +1,9 @@
+$blockHigh: 300px;
```

It's not only High, so rename this var.
```suggestion
$blockSize: 300px;
```